zerver/lib/actions.py

```diff
@@ -785,7 +785,7 @@ def extract_recipients(s):
     except ValueError:
         data = s
 
-    if isinstance(data, six.string_types):
+    if isinstance(data, six.text_type) or isinstance(data, six.binary_type):
```

Comment:
I think I'd rather do a `type: ignore` here and point to: https://github.com/python/typeshed/pull/138, since once that bug is fixed, we're going to want to restore this to the normal, more reasonable code.